targets/ath79-generic

```diff
@@ -266,6 +266,13 @@ device('tp-link-archer-c2-v3', 'tplink_archer-c2-v3', {
 	broken = true,  -- 64M ath9k + ath10k
 })
 
+device('tp-link-archer-c5-v1', 'tplink_archer-c5-v1', {
+	packages = ATH10K_PACKAGES_QCA9880,
+	manifest_aliases = {
+		'archer-c5-v1', -- Upgrade from OpenWrt 19.07
```

Comment:
Unneeded manifest alias. The device was always "tp-link-archer-c5-v1", it never had a different name.

Reply:
Thanks.

Reply:
resolved in 36b201a53a1f1cb9a573687c442b263824e5faee